gax/src/main/java/com/google/api/gax/rpc/ServerStreamingAttemptCallable.java

```diff
@@ -181,15 +180,6 @@ public void cancel() {
     }
     isStarted = true;
 
-    // Propagate the totalTimeout as the overall stream deadline, so long as the user
-    // has not provided a timeout via the ApiCallContext. If they have, retain it.
-    Duration totalTimeout =
-        outerRetryingFuture.getAttemptSettings().getGlobalSettings().getTotalTimeout();
-
-    if (totalTimeout != null && context != null && context.getTimeout() == null) {
-      context = context.withTimeout(totalTimeout);
-    }
```

Comment:
After removing this block, if someone set totalTimeout but not rpc timeout, the stream won't have a deadline?

Reply:
I think this already covered by https://github.com/googleapis/gax-java/pull/1191?